l10n_ch_lsv_dd/model/move_line.py

```diff
@@ -39,3 +39,29 @@ def _is_generate_bvr(self, cr, uid, invoice, context=None):
         return val or (invoice.partner_bank_id and
                        invoice.partner_bank_id.state == 'iban' and
                        invoice.partner_bank_id.lsv_identifier)
+
+    def line2bank(self, cr, uid, ids, payment_mode_id, context=None):
+        ''' Override line2bank to avoid choosing a bank that has only
+            cancelled mandate.
+        '''
+        pay_mode_obj = self.pool.get('payment.mode')
+        if payment_mode_id:

```

Comment:
can you please also give me a pointer to the definition of `suitable_bank_types`